pkg/machine/e2e/init_test.go

```diff
@@ -77,6 +77,20 @@ var _ = Describe("podman machine init", func() {
 		badMemSession, err := mb.setCmd(badMem.withMemory(uint(total))).run()
 		Expect(err).ToNot(HaveOccurred())
 		Expect(badMemSession).To(Exit(125))
+
+		// Check that mounting to certain target directories like /tmp at the / level is NOT ok
+		tmpVol := initMachine{}
+		targetMount := "/tmp"
+		tmpVolSession, err := mb.setCmd(tmpVol.withVolume(fmt.Sprintf("/whatever:%s", targetMount))).run()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(tmpVolSession).To(Exit(125))
```

Comment:
this is failing on WSL because it ignores all volumes so not error is produced.

https://api.cirrus-ci.com/v1/artifact/task/5435362996125696/html/machine-wsl-podman-windows-rootless-host-sqlite.log.html#t--podman-machine-init-bad-init--1